app/locales/en/translations.ts

```diff
@@ -335,17 +335,16 @@ export default {
         contact_title: 'Get in touch',
         contact_technical: 'For emails about technical support:',
         contact_questions: 'For all other questions or comments:',
-        prereg_title: 'Do you have Prereg Challenge related questions?',
-        prereg_paragraph_1: 'Check out our ',
-        prereg_link_text: 'prereg section',
-        prereg_paragraph_2: ' on the cos.io website.',
+        prereg_title: 'Do you have questions about preregistration?',
+        prereg_paragraph_1: 'Check out our',
```

Comment:
I think you accidentally removed a space here.
```suggestion
        prereg_paragraph_1: 'Check out our ',
```